helm/cosmo/charts/studio/values.yaml

```diff
@@ -118,6 +118,8 @@ probes:
 configuration:
   # -- The client ID of the slack app to use for slack app integration
   slackClientId: ""
+  # --- Custom scripts injected into the head of the studio, eg for analytics
```

Comment:
You forgot to run `make docs`. Usually, the CI should catch it.

Reply:
I don't understand?

```make: Nothing to be done for `docs'.```

Reply:
Ahh please use two dashes `--` :smile:

Reply:
Gotcha